app/routes/_gcn.circulars.$circularId.($version)/route.tsx

```diff
@@ -0,0 +1,165 @@
+    useLoaderData<typeof loader>()
+  const searchString = useSearchString()
+
+  const result = useRouteLoaderData<typeof parentLoader>(
+    'routes/_gcn.circulars.$circularId'
+  )
+
+  const latest = version && !result?.history.includes(version)
+  const linkString = `/circulars/${circularId}${
+    !latest && version ? `/${version}` : ''
+  }`
+  return (
+    <>
+      <ButtonGroup>
+        <Link to={`/circulars${searchString}`} className="usa-button">
+          <div className="position-relative">
+            <Icon.ArrowBack
+              role="presentation"
+              className="position-absolute top-0 left-0"
+            />
+          </div>
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Back
+        </Link>
+        <ButtonGroup type="segmented">
+          <Link
+            to={`${linkString}.txt`}
+            className="usa-button usa-button--outline"
+            reloadDocument
+          >
+            Text
+          </Link>
+          <Link
+            to={`${linkString}.json`}
+            className="usa-button usa-button--outline"
+            reloadDocument
+          >
+            JSON
+          </Link>
+          {bibcode ? (
+            <Link
+              to={`https://ui.adsabs.harvard.edu/abs/${bibcode}`}
+              className="usa-button usa-button--outline"
+            >
+              Cite (ADS)
+            </Link>
+          ) : (
+            <Button
+              type="button"
+              disabled
+              outline
+              title="The ADS entry for this Circular is not yet available. ADS entries are updated every week on Monday at 08:00 UTC. Please check back later."
+            >
+              Cite (ADS)
+            </Button>
+          )}
+        </ButtonGroup>
+        {useFeature('CIRCULAR_VERSIONS') && result?.userIsModerator && (
+          <Link
+            to={`/circulars/edit/${circularId}`}
+            className="usa-button usa-button--outline"
+          >
+            Edit
+          </Link>
+        )}
+      </ButtonGroup>
+      <h1 className="margin-bottom-0">GCN Circular {circularId} </h1>
+      {useFeature('CIRCULAR_VERSIONS') && (
+        <CircularsHistory circular={circularId} history={result?.history} />
+      )}
+      <FrontMatter {...frontMatter} />
+      <PlainTextBody className="margin-y-2">{body}</PlainTextBody>
+    </>
+  )
+}
+
+function CircularsHistory({
+  circular,
+  history,
+}: {
+  circular: number
+  history?: number[]
+}) {
+  return (
+    <>
+      <h3 className="margin-y-0">Versions</h3>
+      <div className="margin-bottom-2">
+        <Link to={`/circulars/${circular}`}>Latest</Link>,{' '}
+        {history &&
+          history.map((version) => (
+            <span key={version.toString()}>
+              <Link to={`/circulars/${circular}/${version}`}>
+                Version {version}
+              </Link>
+              ,{' '}
+            </span>
+          ))}
+      </div>
```

Comment:
How about a dropdown menu in the button bar?

Reply:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/5139bd09-9a80-45c0-b7b4-502883477936)

![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/df21abfe-2d4c-450b-beef-02c8f9787670)

Reply:
Let's separate this from the Text/JSON/Cite segmented button. Those are grouped together into a segmented button because they are different formats; the Version dropdown doesn't go with them.

Reply:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/906e1920-b006-4ec5-9433-6196bc71ba01)

Reply:
I like it!

Reply:
Sweet, its in the latest commit!